ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -25,10 +25,10 @@ import com.google.protobuf.{ByteString, Empty}
 
 import scala.util.Try
 
-/** Internal utilities for converting between protobuf messages and our scala
+/** Utilities for converting between protobuf messages and our scala
   * data structures.
   */
-private[kvutils] object Conversions {
+object Conversions {
```

Comment:
Wouldn't `private[state]` suffice? I.e., that way you could use functionality here from all classes under `com.ledger.participant.state`.

Reply:
Yes it would. I'll make that change.